MobileBuy/buy/src/main/java/com/shopify/buy/dataprovider/CustomerServiceDefault.java

```diff
@@ -193,19 +193,18 @@ public void call(CustomerToken token) {
     }
 
     @Override
-    public CancellableTask logoutCustomer(final Callback<Void> callback) {
-        return new CancellableTaskSubscriptionWrapper(logoutCustomer().subscribe(new InternalCallbackSubscriber<>(callback)));
+    public CancellableTask logoutCustomer(Long customerId, final Callback<Void> callback) {
```

Comment:
I agree actually.  We should not need customerId on a lot of these API calls.  We have the token which contains the customerId, and using the one we have already in the token ensures that it is correct.
@bgulanowski @davidmuzi - maybe iOS should be keeping the whole token, then we can clean up these customer APIS, removing the customerId parameter.